wscript

```diff
@@ -77,14 +81,18 @@ def configure(ctx):
     ctx.load('compiler_cxx')
 
     # Check required headers
-    req_headers = ['stdlib.h', 'string.h', 'unistd.h', 'stdint.h', 'stdio.h', 'jpeglib.h']
+    req_headers = ['stdlib.h', 'string.h', 'stdint.h', 'stdio.h']
+    if platform.system() != "Windows":
+        req_headers += ['dlsym.h', 'unistd.h', 'jpeglib.h']
+    else:
+        req_headers += ['windows.h']
```

Comment:
There seems to be a lot of special-casing of windows vs linux in the configure() function. Perhaps it would be cleaner
to split this into a configure_linux() vs configure_win32() and call that from configure()? Some common parts could remain in configure(), but I am fine with some duplication in configure_*() to avoid the interleaved special-casing.

Reply:
Done mostly.